packages/core/admin/ee/server/routes/review-workflows.js

```diff
@@ -80,15 +72,7 @@ module.exports = {
       handler: 'workflows.findById',
       config: {
         middlewares: [enableFeatureMiddleware('review-workflows')],
-        policies: [
-          'admin::isAuthenticatedAdmin',
-          {
-            name: 'admin::hasPermissions',
-            config: {
-              actions: ['admin::review-workflows.read'],
-            },
-          },
-        ],
+        policies: ['admin::isAuthenticatedAdmin'],
```

Comment:
I think we could keep the admin::review-worklfows.read here too, I think the new permissin + this one should be compatible with each other. 
If the user doesnt have permission to read RW settings it should return a 403 error too.